app/lookups/land/device_resolution.rb

```diff
@@ -0,0 +1,9 @@
+module Land
+  class DeviceResolution < ApplicationRecord
+    include TableName
```

Comment:
Relations needed here as well:

```suggestion
    include TableName
    
    has_many :user_agent
```